lib/recipe/bloc/recipe_bloc.dart

```diff
@@ -473,18 +580,17 @@ class RecipeBloc extends Bloc<RecipeEvent, RecipeState> {
     emit(state.copyWith(recipes: recipes));
   }
 
-  void _onIngredientDescriptionChanged(
-      IngredientDescriptionChanged event, Emitter<RecipeState> emit) {
+  void _onIngredientOtherNameChanged(
+      IngredientOtherNameChanged event, Emitter<RecipeState> emit) {
     List<Recipe> recipes = List.from(state.recipes);
 
     int changedRecipeIndex = recipes.indexOf(event.recipe);
     List<Ingredient> ingredients =
         List.from(recipes[changedRecipeIndex].ingredients);
     int changedIngredientIndex = ingredients.indexOf(event.ingredient);
 
-    Ingredient ingredient = ingredients[changedIngredientIndex].copyWith(
-        description: Description.dirty(event.ingredientDescription),
-        saved: false);
+    Ingredient ingredient = ingredients[changedIngredientIndex]
+        .copyWith(otherName: event.ingredientOtherName, saved: false);
```

Comment:
yeah, I think we should think about a more expressive name for this instead of `otherName`.